openedx/features/content_type_gating/migrations/0001_initial.py

```diff
@@ -2,6 +2,7 @@
 # Generated by Django 1.11.16 on 2018-11-08 19:43
 from __future__ import unicode_literals
 
+from __future__ import absolute_import
```

Comment:
Looks like you ended up with split `__future__` imports on these migration files.

Reply:
Fixed.